game-app/game-core/src/main/java/games/strategy/engine/framework/ui/SaveGameFileChooser.java

```diff
@@ -45,12 +45,13 @@ private static String getSaveGameName(final GameData gameData) {
   }
 
   private static String formatGameName(final String gameName) {
+    @SuppressWarnings("JavaTimeDefaultTimeZone")
```

Comment:
K, adapted it.
The reason why I added it was just to try to be able to finally compile everything successfully. I spent several hours to fix code just to be able to compile all code once. Therefore, I did not check each warning I needed to suppress, but I also thought about running a check later with disabled suppress warnings to find them back and really fix them.

Reply:
What command are you running to compile?

This is odd to see.
1. The build does pass, this should not be OS specific.
2. We removed error prone, you should not be encountering these. Which branch & git SHA are you building?

error prone was removed in: commit 337274a8d82c7ae8ec405fb65f1b780d5a6f9df9 (tag: 2.6.372)

Reply:
Check the current failed runs that happened after I just changed a comment.
Before all checks passed successfully.

Reply:
It turns out the PR build is rebasing PRs against latest master.

Following the commands in the 'checkout' sequence, I was able to replicate the build errors.

Notice for example these lines:

> /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +4ee61ab2062420c10cc2f1faf8b938b4084711e7:refs/remotes/pull/9939/merge
      
> * [new ref]         4ee61ab2062420c10cc2f1faf8b938b4084711e7 -> pull/9939/merge
       
> /usr/bin/git checkout --progress --force refs/remotes/pull/9939/merge
   Note: switching to 'refs/remotes/pull/9939/merge'.
  
> HEAD is now at 4ee61ab Merge fc03b60f7e2e02c7402c9f1afad530434dafaef6 into c61e13cd2d87b288446a8c00aa9feedab7cbaf4b
  
That last one is really key, fc03b6 is the head of the PR branch, c61e13cd is the head of master. After getting to that point a `./gradlew compileJava` replicates the errors.

----

If you merge latest master into your branch you should see the same errors. Though, a merge of latest master will result in merge conflicts, you'll need to resolve those.

  

What partially clued me in is that `HttpInteractionException` is removed and so is the custom `FeignDecoder` is as well: 
- https://github.com/triplea-game/triplea/pull/9933/files#diff-a66634c3205f49af440b11dbf069687dc307f6fd1e2b0ddb1a3077031d523fb7L107
- https://github.com/triplea-game/triplea/pull/9933/files#diff-854e28d85dbe0561b51afd0075f01be7451ea6113e1fe0c84649413d4c25cea9

Though, those were recent updates - it was odd to see an error message relating to that on a seemingly older branch that would not have had those changes in it.

The fact we merge PRs onto latest master is actually quite good. In this case had the build passed we would have merged to then later see an error in that build. Instead we see it now, though it's confusing where those errors come from. Of note, it looks like the build in your fork worked fine on this step (notably becuase it was rebasing master (frigoref/master) onto itself, and is not using the upstream version 'triplea-game/master').

Side-note: when working with forks it's generally easiest to avoid using 'master'. 'master' essentially kept in sync with upstream master, eg: `git checkout master; git fetch upstream; git pull --rebase upstream/master; git push origin master` Then feature branches are cut from 'master' with `git checkout master; git checkout -b branch-name/some-branch-name`.